src/components/Atoms/Input/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useRef, useEffect } from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color.js";
+import FONTFAMILY from "../../../variables/font_family.js";
+import TEXT from "../../../variables/texts.js";
+
+const Input = ({ defaultValue = "", onEditComplete }) => {
+  const inputRef = useRef(null);
+
+  useEffect(() => {
+    inputRef.current.value = defaultValue;
+    inputRef.current.focus();
+    inputRef.current.onblur = (e) => onEditComplete(e.target.value);
+    inputRef.current.onkeydown = (e) => {
+      if (e.key === "Enter") {
+        onEditComplete(e.target.value);
+      }
+    };
+  }, []);
+  return <StyledInput ref={inputRef}></StyledInput>;
```

Comment:
```suggestion
  return <StyledInput ref={inputRef} />;
```
中に何も取らないときはこのように書くとスッキリします